.github/dependabot.yml

```diff
@@ -4,17 +4,22 @@
 
 version: 2
 updates:
-    # --- GitHub Actions
-    - package-ecosystem: 'github-actions'
-      directory: '/'
-      schedule:
-          interval: 'daily'
+  # --- GitHub Actions
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+        interval: 'daily'
 
-    - package-ecosystem: 'npm'
-      directory: '/'
-      open-pull-requests-limit: 20
-      rebase-strategy: 'auto'
-      schedule:
-          interval: 'weekly'
-      allow:
-          - dependency-type: 'direct'
+  - package-ecosystem: 'npm'
+    directory: '/'
+    open-pull-requests-limit: 20
+    rebase-strategy: 'auto'
+    schedule:
+        interval: 'weekly'
+    allow:
+        - dependency-type: 'direct'
+    ignore:
+        - dependency-name: 'typescript'
```

Comment:
I'm fine with new versions of TS

Reply:
OK. I will remove the ignore entry for typescript.